htdocs/core/modules/mailings/thirdparties.modules.php

```diff
@@ -29,7 +29,7 @@ class mailing_thirdparties extends MailingTargets
 	public $desc = "Third parties (by categories)";
 	public $require_admin = 0;
 
-	public $require_module = array("societe"); // This module allows to select by categories must be also enabled if category module is not activated
+	public $require_module = array("societe", "category"); // This module allows to select by categories must be also enabled if category module is not activated
```

Comment:
When category module is not enabled, the selector must still be available. Only the filter on categories must be disabled.

Reply:
Indeed you're right, I corrected that